Libraries/LibJS/CMakeLists.txt

```diff
@@ -248,5 +248,11 @@ if("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "i.86.*")
     target_link_libraries(LibJS PRIVATE atomic)
 endif()
 
+if (WIN32)
```

Comment:
While not supported, it is possible to use different compilers to build for Win32 target. It should be better to check for Clang-CL toolchain instead of just checking for WIN32. You can check for clang-cl by checking if `CMAKE_CXX_COMPILER_ID` is set to `Clang` and `CMAKE_CXX_COMPILER_FRONTEND_VARIANT` is set to `MSVC` at the same time.

Reply:
MSVC does not support options setting the frame pointer in 64 bit mode. If we indeed decide just not to set any option here, this would be the correct behavior. If we decide to set an option for clang-cl, then I'll change it to check for clang-cl. Even though I doubt cl would just work.